fuse_core/CMakeLists.txt

```diff
@@ -1,519 +1,500 @@
+#     test_throttled_callback
+#     test/throttled_callback.test
+#     test/test_throttled_callback.cpp
+#   )
+#   target_link_libraries(test_throttled_callback
+#     ${PROJECT_NAME}
+#     ${catkin_LIBRARIES}
+#   )
+#   set_target_properties(test_throttled_callback
+#     PROPERTIES
+#       CXX_STANDARD 14
+#       CXX_STANDARD_REQUIRED YES
+#   )
+
+#   # Util tests
+#   catkin_add_gtest(test_util
+#     test/test_util.cpp
+#   )
+#   add_dependencies(test_util
+#     ${catkin_EXPORTED_TARGETS}
+#   )
+#   target_include_directories(test_util
+#     PRIVATE
+#       include
+#       ${Boost_INCLUDE_DIRS}
+#       ${catkin_INCLUDE_DIRS}
+#       ${CERES_INCLUDE_DIRS}
+#       ${EIGEN3_INCLUDE_DIRS}
+#   )
+#   target_link_libraries(test_util
+#     ${catkin_LIBRARIES}
+#   )
+#   set_target_properties(test_util
+#     PROPERTIES
+#       CXX_STANDARD 14
+#       CXX_STANDARD_REQUIRED YES
+#   )
+
+#   # UUID tests
+#   catkin_add_gtest(test_uuid
+#     test/test_uuid.cpp
+#   )
+#   add_dependencies(test_uuid
+#     ${catkin_EXPORTED_TARGETS}
+#   )
+#   target_include_directories(test_uuid
+#     PRIVATE
+#       include
+#       ${Boost_INCLUDE_DIRS}
+#       ${catkin_INCLUDE_DIRS}
+#       ${CERES_INCLUDE_DIRS}
+#       ${EIGEN3_INCLUDE_DIRS}
+#   )
+#   target_link_libraries(test_uuid
+#     ${PROJECT_NAME}
+#     ${catkin_LIBRARIES}
+#   )
+#   set_target_properties(test_uuid
+#     PROPERTIES
+#       CXX_STANDARD 14
+#       CXX_STANDARD_REQUIRED YES
+#   )
+
+#   # Variable tests
+#   catkin_add_gtest(test_variable
+#     test/test_variable.cpp
+#   )
+#   add_dependencies(test_variable
+#     ${catkin_EXPORTED_TARGETS}
+#   )
+#   target_include_directories(test_variable
+#     PRIVATE
+#       include
+#       ${Boost_INCLUDE_DIRS}
+#       ${catkin_INCLUDE_DIRS}
+#       ${CERES_INCLUDE_DIRS}
+#       ${CMAKE_CURRENT_SOURCE_DIR}
+#       ${EIGEN3_INCLUDE_DIRS}
+#   )
+#   target_link_libraries(test_variable
+#     ${PROJECT_NAME}
+#   )
+#   set_target_properties(test_variable
+#     PROPERTIES
+#       CXX_STANDARD 14
+#       CXX_STANDARD_REQUIRED YES
+#   )
+# endif()
 
 #############
 ## Install ##
 #############
 
-install(
-  TARGETS ${PROJECT_NAME}
-  ARCHIVE DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
-  LIBRARY DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
-  RUNTIME DESTINATION ${CATKIN_GLOBAL_BIN_DESTINATION}
+install(TARGETS ${PROJECT_NAME} EXPORT ${PROJECT_NAME}-export
+  ARCHIVE DESTINATION lib
+  LIBRARY DESTINATION lib
+  RUNTIME DESTINATION bin
 )
 
 install(
   TARGETS fuse_echo
-  RUNTIME DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}
+  DESTINATION lib/${PROJECT_NAME}
 )
 
-install(
-  DIRECTORY include/${PROJECT_NAME}/
-  DESTINATION ${CATKIN_PACKAGE_INCLUDE_DESTINATION}
+install(DIRECTORY include/
+  DESTINATION include/${PROJECT_NAME}
 )
 
-#############
-## Testing ##
-#############
+ament_export_targets(${PROJECT_NAME}-export HAS_LIBRARY_TARGET)
+ament_export_dependencies(${ament_dependencies})
```

Comment:
It looks like this variable is undefined (context: https://github.com/locusrobotics/fuse/pull/281#discussion_r991624341 )

Reply:
Oops, fixed: aa287f03c254134f577e887a6f9173a2d0b05619